index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style/style.css" type="text/css">
+        <title>YouTube</title>
+    </head>
+    <body>
+        <!--Start of navigation bar-->
+        <header>
+            <nav>
+                <a href="#"><img id="bars" class="img" src="img/img_bar.png" alt="image bar"></a>
+                <a href="#"><img id="logo" src="img/youtube_logo.png" alt="YouTube"></a>
+                <input id="search" type="text" placeholder="Search" class="img">
+                <a href="#"><button id="loop"><img id="img1" src="img/img_loop.png" alt="image loop"></button></a>
```

Comment:
Here you could use a font awesome icon on this button, instead of an image (line 16)